plugins/vpc-shared-eni/plugin/commands.go

```diff
@@ -169,5 +169,12 @@ func (plugin *Plugin) Del(args *cniSkel.CmdArgs) error {
 		log.Errorf("Failed to delete endpoint, ignoring: %v", err)
 	}
 
+	if netConfig.DeleteNetwork {
```

Comment:
I assume we need this to cleanup the network for Windows task ENIs only? Is this the only task ENI-specific behavior change you are going to introduce? If there are more, I am wondering whether it is better to create a "TaskENI" property rather than creating a new property like "DeleteNetwork" for each such behavior change.

Reply:
Yes, this is the only task-ENI specific behavior change which will be introduced.

However, given that we might need a few more changes for ecs-bridge, we can consider creating a more generic property such as 'TaskNetworking' which can encompass all such changes.